engine/index.js

```diff
@@ -359,6 +360,31 @@ function checkForNewData(features, dbTestNumber) {
   );
 }
 
+function sendNotifications(features, dbNumber) {
+  return notifications.setClient(dbNumber)
+  .then(() => Promise.all(features.map(feature => {
+    if (feature.just_started) {
+      return notifications.sendNotifications(feature.slug, {
+        feature: feature.slug,
+        message: 'started watching',
+      }, true, dbNumber);
```

Comment:
Can we avoid passing the `dbNumber`? We're already passing it earlier with `notifications.setClient`.